compiler-lib/compiler/commands/test.rkt

```diff
@@ -666,7 +666,13 @@
   ;; This should be in Racket somewhere and return all the collection
   ;; paths, rather than just the first as collection-path does.
   (define (collection-paths c)
-    (match-define (list-rest sc more) (map path->string (explode-path c)))
+    (define (path->string* maybe-path)
```

Comment:
OH, you mean add this function to the standard library. That sounds good.

It seems like that should go in `racket/path`, or should it go with the main racket pathing functions?